modules/repository/file_test.go

```diff
@@ -1,8 +1,7 @@
-// Copyright 2017 The Gitea Authors. All rights reserved.
-// Copyright 2014 The Gogs Authors. All rights reserved.
+// Copyright 2023 The Gitea Authors. All rights reserved.
 // SPDX-License-Identifier: MIT
 
-package repo
+package repository
```

Comment:
I don't think we need to update license when move a file.

Reply:
The blame indicates the whole code in this file is created reasently by gitea maintainers ... so i dont mid to correct the license headrr